### PR TITLE
docs: populate readme

### PR DESCRIPTION
**Pull request by Khuzaima05**
### Description

Update ReadMe.md and setting.syml

###issue: https://github.com/terraform-ibm-modules/terraform-ibm-secrets-manager-secret-group/issues/3
#### Changes that affect the core Terraform module or submodules
- [ ] Bug fix
- [ ] New feature
- [ ] Dependency update

#### Changes that don't affect the core Terraform module or submodules
- [ ] Examples or tests (addition or updates of examples or tests)
- [x] Documentation update
- [ ] CI-related update (pipeline, etc.)
- [ ] Other

#### Release required?
Identify the type of release. For information about the changes in a semantic versioning release, see [Release versioning](https://terraform-ibm-modules.github.io/documentation/#/versioning).

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

##### Release notes content

If a release is required, replace this text with information that users need to know about the release. Write the release notes to help users understand the changes, and include information about how to update from the previous version.

Your notes help the merger write the commit message for the PR that is published in the release notes for the module.

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### Merge actions for mergers

- Use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents and any release notes provided by the PR author. The commit message determines whether a new version of the module is needed, and if so, which semver increment to use (major, minor, or patch).
- Merge by using "Squash and merge".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,148 +1,55 @@
...
-<!-- BEGIN TOC HOOK -->
-Table of Contents
-====================
 
-* [Terraform modules](#Terraform-modules)
-    * [Terraform Modules Template Project](#Terraform-Modules-Template-Project)
-* [Examples](#Examples)
-* [Contributing](#Contributing)
-<!-- END TOC HOOK -->
+This module creates a secret in an IBM Secrets Manager secrets group.
```
Comment (ocofaigh) [inline README.md:10]: This is not correct. This module creates a secret group, nota secret

File: README.md [hunk 2]
```diff
@@ -1,148 +1,55 @@
...
 
-<!-- Remove the content in this H2 heading after completing the steps -->
+The module supports the following secret types:
 
-## Submit a new module
+- [Arbitrary](https://cloud.ibm.com/docs/secrets-manager?topic=secrets-manager-arbitrary-secrets&interface=ui)
+- [User credentials](https://cloud.ibm.com/docs/secrets-manager?topic=secrets-manager-user-credentials&interface=ui)
+- [Imported Certificate](https://cloud.ibm.com/docs/secrets-manager?topic=secrets-manager-certificates&interface=api#import-certificates)
```
Comment (ocofaigh) [inline README.md:16]: None of this is correct. This module creates a secret group, it doesn't create secrets

File: README.md (hunk 1 above)
Comment (Khuzaima05) [reply README.md]: resolved

File: README.md (hunk 2 above)
Comment (Khuzaima05) [reply README.md]: Resolved

File: README.md [hunk 3]
```diff
@@ -1,148 +1,49 @@
...
-<!-- Remove the content in this previous H2 heading -->
-## Reference architectures
-
-<!--
-Add links to any reference architectures for this module.
-(Usually in the `/reference-architectures` directory.)
-See "Reference architecture" in Authoring Guidelines in the public documentation at
-https://terraform-ibm-modules.github.io/documentation/#/implementation-guidelines?id=reference-architecture
--->
+This module creates a secret group in an IBM Secrets Manager secrets group.
```
Comment (ocofaigh) [inline README.md:10]: IBM Secrets Manager secrets ~group~ instance.

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**